tests/test_commands.py

```diff
@@ -109,18 +111,27 @@ def test_create_item(self):
                     self.assertEqual(item.id, item_id)
 
                     # uncomment these lines to update the expected_output files
-                    # import shutil
-                    # shutil.copyfile(os.path.join(tmp_dir, fname),
-                    # f"{granule_href}/expected_output.json")
+                    import shutil
+
+                    shutil.copyfile(
+                        os.path.join(tmp_dir, fname),
+                        f"{granule_href}/expected_output.json",
+                    )
 
                     def mk_comparable(i: pystac.Item) -> Dict[str, Any]:
                         d = i.to_dict(include_self_link=False)
                         for a in d["assets"].values():
                             a["href"] = a["href"].split("data-files")[1]
 
-                        for c in d["geometry"]["coordinates"][0]:
-                            c[0] = round(c[0], 5)
-                            c[1] = round(c[1], 5)
+                        if len(d["geometry"]["coordinates"]) > 1:
+                            for i in range(0, len(d["geometry"]["coordinates"][0])):
+                                for c in d["geometry"]["coordinates"][0][i]:
+                                    c[0] = round(c[0], 5)
```

Comment:
In the stac.py file `bbox` and `proj_bbox` are rounded to 6 decimal places, while these are 5.
Recommend creating a constant in one of the modules for lat/lon precision and reference that where it is hardcoded.